sockeye/data_io.py

```diff
@@ -396,79 +396,47 @@ def statistics(self):
                               average_len_target_per_bucket=self.mean_len_target_per_bucket,
                               length_ratio_stats_per_bucket=self.length_ratio_stats_per_bucket)
 
-
-def shard_data(source_fnames: List[str],
-               target_fnames: List[str],
-               source_vocabs: List[vocab.Vocab],
-               target_vocabs: List[vocab.Vocab],
-               num_shards: int,
-               buckets: List[Tuple[int, int]],
-               length_ratio_mean: float,
-               length_ratio_std: float,
-               output_prefix: str):
+def create_shards(source_fnames: List[str],
+                  target_fnames: List[str],
+                  num_shards: int,
+                  output_prefix: str):
     """
-    Assign int-coded source/target sentence pairs to shards at random.
+    Assign source/target sentence pairs to shards at random.
 
     :param source_fnames: The path to the source text (and optional token-parallel factor files).
     :param target_fnames: The path to the target text (and optional token-parallel factor files).
-    :param source_vocabs: Source vocabulary (and optional source factor vocabularies).
-    :param target_vocabs: Target vocabulary (and optional target factor vocabularies).
     :param num_shards: The total number of shards.
-    :param buckets: Bucket list.
-    :param length_ratio_mean: Mean length ratio.
-    :param length_ratio_std: Standard deviation of length ratios.
     :param output_prefix: The prefix under which the shard files will be created.
-    :return: Tuple of source (and source factor) file names, target (and target factor) file names
-             and statistics for each shard, as well as global statistics.
+    :return: Tuple of source (and source factor) file names and target (and target factor) file names for each shard.
     """
     os.makedirs(output_prefix, exist_ok=True)
```

Comment:
Could we support the special case of `num_shards=1` by returning the paths of the original files?

Reply:
Thanks!